projects/vtk.cmake

```diff
@@ -32,5 +38,5 @@ superbuild_add_project(vtk
     -DVTK_MODULE_ENABLE_VTK_RenderingOpenGL2=YES
     -DVTK_MODULE_ENABLE_VTK_RenderingVolumeOpenGL2=YES
     -DVTK_MODULE_ENABLE_VTK_TestingCore=YES
-    -DVTK_MODULE_ENABLE_VTK_RenderingRayTracing=YES
+    -DVTK_MODULE_ENABLE_VTK_RenderingRayTracing:STRING=${vtk_raytracing_enabled}
```

Comment:
Why not testing (ospray_enabled) directly?

Reply:
looks above, this is a string, not a bool